<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Dominic D. G. Owens</title>
    <link rel="stylesheet" href="styles.css">
    <link rel="icon" href="favicon.ico" type="image/x-icon">
</head>
<body>
    <header>
        <div class="logo">Dominic D. G. Owens</div>
        <nav>
            <ul>
                <li><a href="#about">About Me</a></li>
                <li><a href="#skills">Skills</a></li>
                <li><a href="#publications">Publications</a></li>
            </ul>
        </nav>
    </header>

    <section id="hero">
        <img src="headshot.jpg" alt="Your Headshot" class="headshot">
        <h1>Dr. Dominic D. G. Owens</h1>
        <h2>Senior Bioinformatician | Amphista Therapeutics</h2>
        <p>Postdoc | Structural Genomics Consortium, University of Toronto <br> PhD (DPhil) | Weatherall Institute of Molecular Medicine, University of Oxford</p>
        <div class="button-group">
            <a href="mailto:dominic.d.g.owens@gmail.com" class="btn" target="_blank">Email</a>
            <a href="DominicOwens_CV.pdf" class="btn" target="_blank">CV</a>
            <a href="https://github.com/d0minicO" class="btn" target="_blank">Github</a>
            <a href="https://www.linkedin.com/in/dominic-owens/" class="btn" target="_blank">LinkedIn</a>
        </div>
        <div class="button-group">
            <a href="https://scholar.google.co.uk/citations?user=oEsW6BEAAAAJ&hl=en" class="btn" target="_blank">Google Scholar</a>
            <a href="DO_thesis.pdf" class="btn" target="_blank">PhD Thesis</a>
        </div>
    </section>
    

    <section id="about">
        <img src="long_photo.jpg" alt="About Photo" class="about-photo">
        <div class="about-text">
            <h2>About Me</h2>
            <p>I am a highly motivated bioinformatician with 10 years of experience extracting key insights from biological data and communicating findings to industry and academic stakeholders through compelling visuals. My background as a bench scientist combined with expertise in drug discovery, epigenetics, and disease mechanisms, enables me to derive insights and contextualise them within the broader biological context. With proficiency in machine learning and statistics, I deliver actionable insights that drive real-world impact.</p>
        </div>
    </section>

    <section id="skills">
        <h2>Key Skills</h2>
        <ul>
            <li>
                <img src="viz.png" alt="Omics Data Analysis" class="skill-image">
                <div class="skill-text">
                    <p><strong>Omics Data Analysis</strong></p>
                    <p>Proficient in analyzing and interpreting large-scale omics datasets, including genomics, transcriptomics, and proteomics, to extract meaningful biological insights.</p>
                </div>
            </li>
            <li>
                <img src="ml.png" alt="Machine Learning and Statistics" class="skill-image">
                <div class="skill-text">
                    <p><strong>Machine Learning and Statistics</strong></p>
                    <p>Experienced in applying machine learning algorithms and statistical techniques to solve complex biological problems and generate predictive models.</p>
                </div>
            </li>
            <li>
                <img src="omics.png" alt="Data Visualisation" class="skill-image">
                <div class="skill-text">
                    <p><strong>Data Visualisation</strong></p>
                    <p>Skilled in creating clear and informative visualizations to effectively communicate data findings and insights to diverse audiences.</p>
                </div>
            </li>
        </ul>
    </section>

    <section id="publications">
        <h2>Selected Publications</h2>
        <ul>
            <li>
                Hughes, S.J., et al. <a href="https://www.biorxiv.org/content/10.1101/2024.12.31.630899v1" target="_blank">Selective degradation of BRD9 by a DCAF16-recruiting targeted glue: mode of action elucidation and in vivo proof of concept</a> BioRXiv (2025).
                <ul>
                    <br>
                    <li>I contributed proteomics data analysis from raw files, statistical analysis, and data visualisation on this exciting targeted protein degradation project with talented colleagues at Amphista Therapeutics.</li>
                </ul>
            </li>
            <br>
            <li>
                <strong>Owens, D.D.G.</strong>, Maitland, M.E.R., et al. <a href="https://doi.org/10.1038/s41589-024-01618-0" target="_blank">A chemical probe to modulate human GID4 Pro/N-degron interactions</a> Nat Chem Biol (2024).
                <ul>
                    <br>
                    <li>I was first author/project lead on this exciting project while at SGC Toronto. We identified and characterised a novel E3 ligase inhibitor, <a href="https://www.thesgc.org/chemical-probes/PFI-7" target="_blank">PFI-7</a>, that prevents substrates binding to the CTLH/GID complex via substrate receptor GID4. I analysed proteomics data and proximity-dependent biotinylation (BioID) data, <a href="https://github.com/d0minicO/GID4_analysis" target="_blank">published analysis code</a>, conducted cell biology experiments, and generated the figures and wrote the paper.</li>
                </ul>
            </li>
            <br>
            <li>
                Nie, D.Y., Tabor, J.R., et al. <a href="https://doi.org/10.1038/s41589-024-01660-y" target="_blank">Recruitment of FBXO22 for targeted degradation of NSD2</a>. Nat Chem Biol (2024). 
                <ul>
                    <br>
                    <li>I was co-author on this impressive story from SGC Toronto uncovering FBXO22 as a tractable E3 ligase for Targeted Protein Degradation (TPD). I analysed BioID data, generated data visualisations, and <a href="https://doi.org/10.5281/zenodo.10930672" target="_blank">published analysis code</a>.</li>
                </ul>
            </li>
            <br>
            <li>
                Harding, R. J., et al. <a href="https://doi.org/10.1021/acs.jmedchem.3c00314" target="_blank">Discovery and characterization of a chemical probe targeting the zinc-finger ubiquitin-binding domain of HDAC6</a>. J Med Chem (2023).
                <ul>
                    <br>
                    <li>I contributed to this exciting hit finding project by performing chemoproteomics and cell biology experiments to characterise a novel ubiquitin-binding domain inhibitor of HDAC6.</li>
                </ul>
            </li>
            <br>
            <li>
                Dilworth, D., et al. <a href="https://doi.org/10.1038/s41589-021-00898-0" target="_blank">A chemical probe targeting the PWWP domain alters NSD2 nucleolar localization</a>. Nat Chem Biol (2022).
                <ul>
                    <br>
                    <li>This was the first publication I contributed to during my postdoc. I demonstrated using chromatin immunoprecipitation in multiple myeloma cells that the novel inhibitor of the histone lysine methyltransferase NSD2, that targets its' PWWP chromatin reader domain, does not impact histone 3 lysine 36 dimethylation (H3K36me2).</li>
                </ul>
            </li>
            <br>
            <li>
                <strong>Owens, D.D.G.</strong>, et al. <a href="https://doi.org/10.1038/s41467-022-28376-8" target="_blank">Dynamic Runx1 chromatin boundaries affect gene expression in hematopoietic development</a>. Nat Commun (2022).
                <ul>
                    <br>
                    <li>In the main publication from my PhD, I used advanced chromosome conformation capture DNA sequencing technology to discover that the gene Runx1--also known as AML1 owing to its' role in causing human leukaemia--is subdivided into distinct chromatin domains that are associated with promoter activation during blood cell development. I performed the wet lab experiments, analysed sequencing data, generated figures, wrote the paper, and <a href="https://doi.org/10.5281/zenodo.5781832" target="_blank">published analysis code</a>.</li>
                </ul>
            </li>
            <br>
            <li>
                <strong>Owens, D.D.G.</strong>, et al. <a href="https://doi.org/10.1093/nar/gkz459" target="_blank">Microhomologies are prevalent at Cas9-induced larger deletions</a>. Nucleic Acids Res (2019).
                <ul>
                    <br>
                    <li>During the course of my studies, I (inadvertently!) discovered that CRISPR/Cas9 genome editing frequently results in larger-than-anticipated on-target deletions that frequently go undetected by common genotyping strategies. Using droplet digital PCR and DNA sequencing, I characterised the nature and frequency of these larger deletions, discovering that short microhomologous DNA sequences often flank cut sites. I performed the experiments, analysed the data, and published an R package called <a href="https://github.com/d0minicO/mhscanR" target="_blank">mhscanR</a> for analysing microhomologies at CRISPR/Cas9 cut sites. </li>
                </ul>
            </li>
            <br>
            
        </ul>
    </section>

    <footer>
        <p>&copy; 2024 Dominic D. G. Owens. All rights reserved.</p>
        <div class="social-media">
            <script type="text/javascript" src="//counter.websiteout.com/js/9/0/0/0"></script>
        </div>
    </footer>
</body>
</html>
